who-metrics-ui/src/components/RepositoriesTable.tsx

```diff
@@ -405,6 +412,16 @@ const getComparator = (sortColumn: keyof Repo): Comparator => {
           .toLowerCase()
           .localeCompare(b[sortColumn].toLowerCase());
       };
+    
+    // Multi option, alphabetical
+    case 'topics':
+      return (a, b) => {
+        const first = a[sortColumn].sort()[0]
+        const second = b[sortColumn].sort()[0]
+        if (second === undefined) return -1;
+        if (first === undefined) return 1;
```

Comment:
👍 - I tend to default to the explicit in TS because I'm worried I'll miss a case (i.e. `0` being false-y), but agree in this case implicit makes sense.